.env.example

```diff
@@ -0,0 +1,4 @@
+NODE_ENV=development
```

Comment:
Above each line, please add a comment

```
# NODE_ENV should be one of "development" or "production"
NODE_ENV=development

# PORT is the port used by the web server
PORT=8080

...
```

This example file can serve as both a template, and documentation.

Reply:
- renamed .env.example to env.example
- added instructions in contributing.md
- added comments in env.example